opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -67,7 +67,7 @@
 _DEFAULT_OTEL_LINK_ATTRIBUTE_COUNT_LIMIT = 128
 
 
-_ENV_VALUE_UNSET = "unset"
```

Comment:
didn't we use `unset` so users should define it explicitly and can avoid the sudden surprise when not setting env either by no knowledge/ignorance considered as unlimited?

Reply:
We did but spec recommends using empty value as unlimited.